### ISSUE TITLE
[Selective Sync] Push default selective sync policy to client?

### ISSUE THREAD

**Issue by mjdemilliano** [owncloud/client#2517]
I am extremely happy with the new Selective Sync feature, so thanks for this big improvement!

However, I am wondering if it is possible to push a default selective sync policy to the client, somehow?

What I would like to achieve is that users which are invited to a shared folder do not automatically download this shared folder to their computer first, then decide they do not want this and then have to go to their Selective Sync configuration and turn this off.

So I would like to do the following: I share a folder "Projects" with users, containing subfolders "Project A", "Project B" and so on. All new folders should not be synchronized to clients by default. Users can see which projects there are, and then decide to want to sync one of them, e.g. "Project B" because they are actively working on it.

Typical application would be that there are 100 folders under "Projects" of which only a few are relevant and need to be synced.

**Comment by MTRichards** [comment]
https://github.com/owncloud/enterprise/issues/558

**Comment by jancborchardt** [comment]
This doesn’t need to be an option. New shares should never automatically be synced before they are accepted.

On the server side we will introduce a page for pending shares: https://github.com/owncloud/core/issues/13495
The client then also needs to have a mechanism to notify of new shares and the ability to accept them (list them up top in the activity stream with an accept/reject button).

**Comment by MTRichards** [comment]
Same for mobile of course too. We need to look at the architecture for these sorts of communications.

**Comment by jancborchardt** [comment]
Yep, the mobile apps would have it in the »Transfers« view, up top.

**Comment by MTRichards** [comment]
Ok. Good for the roadmap conversation.

**Comment by pierrejochem** [comment]
This feature would be a big benefit. Especially with external storages this is a pain in the a.. if they are automatically downloaded from the server.

**Comment by michaelstingl** [comment]
1st step is https://github.com/owncloud/client/pull/6761
